src/AppStore.ts

```diff
@@ -0,0 +1,23 @@
+import {computed, observable} from "mobx";
+import {initializeAPIClients} from "./config/config";
+import * as _ from 'lodash';
+
+export class AppStore {
```

Comment:
We need an "AppStore" because configuration object is not observable and probably shouldn't be as it would not be expected to update I don't think.

Reply:
should we call it `AuthStore` to reflect it's relation to authentication, or is there a plan to extend its purpose?